dev/deps/spark-deps-hadoop-2.7-hive-2.3

```diff
@@ -125,6 +126,7 @@ jakarta.ws.rs-api/2.1.6//jakarta.ws.rs-api-2.1.6.jar
 jakarta.xml.bind-api/2.3.2//jakarta.xml.bind-api-2.3.2.jar
 janino/3.0.16//janino-3.0.16.jar
 javassist/3.25.0-GA//javassist-3.25.0-GA.jar
+javax.annotation-api/1.3.2//javax.annotation-api-1.3.2.jar
```

Comment:
I think we do not need `javax.annotation-api-1.3.2.jar`. We do not need build Avro.

Reply:
excluded now as suggested for avro-ipc-jetty too